infrastructure/platform/main.tf

```diff
@@ -0,0 +1,33 @@
+resource "kubernetes_namespace" "live" {
```

Comment:
What's the reason for choosing the `live` resource name?

Reply:
to distinguish from `paper` but we could call it whatever, essentially is production